lib/cancogen/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.7'
+
+services:
+  cancogen_dashboard:
+    build:
+      context: $PWD/lib/cancogen
+
+    image: ${DOCKER_REGISTRY}/cancogen:${CANCOGEN_DASHBOARD_VERSION:-latest}
+    volumes: 
+      - '.:/cancogen_dashboard'
```

Comment:
use volumes instead of local file paths see Makefile docker-volumes method

Reply:
I did some reading into the volumes specifically for React apps and this was only being done to allow hot reloading for development, so I'm removing this section for our deployment purposes